packages/@aws-cdk/aws-codepipeline/lib/pipeline.ts

```diff
@@ -618,7 +606,7 @@ export class Pipeline extends PipelineBase {
  * the cross-region capabilities of CodePipeline.
  * You get instances of this interface from the {@link Pipeline#crossRegionSupport} property.
  */
-export interface CrossRegionSupport {
```

Comment:
Just add @experimental. There are people who might depend on this already

Reply:
Done.